domenetjenester/fpsak/pom.xml

```diff
@@ -41,6 +41,15 @@
             <groupId>io.dropwizard.metrics</groupId>
             <artifactId>metrics-annotation</artifactId>
         </dependency>
+        <dependency>
+            <groupId>no.finn.unleash</groupId>
+            <artifactId>unleash-client-java</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
```

Comment:
Bruker ikke vi jackson allerede? Gjør bibliotekene forskjellige ting?

Reply:
Ja jackson, blant andre tror jeg. Nei, de produserer og parser json begge to. Et eksempel jeg så på SO brukte org.json, så da kopierte jeg bare det.

Reply:
OK, ikke ideelt at vi bruker to forskjellige - men men